src/components/TradeWidget/index.tsx

```diff
@@ -70,10 +98,24 @@ const TradeWidget: React.FC = () => {
     }
   }
 
+  /**
+   * Regardless of the validation, check the current input values
+   * Returns true if both are valid numbers > 0
+   */
+  function showOrderDetails(): boolean {
```

Comment:
memoize

Reply:
Don't think I can memoize here because the values come from [watch](https://react-hook-form.com/api#watch) that work like instance variables AFAIU. 
So it's not possible to use those values as dependencies on [useMemo](https://reactjs.org/docs/hooks-reference.html#usememo).
Unless I'm mistaken?